Bottom: 34e9c9bb8eac8b5ca92daf4ff5715d0599f6d82e
Top:    86aba3185d95e209ea12c017663afbc8fbb46c12
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 13:51:51 +0000

Refresh of more-play-with-quad-edge-data

---

diff --git a/src/hid/step/object3d_step.c b/src/hid/step/object3d_step.c
index 24082a4..449d44a 100644
--- a/src/hid/step/object3d_step.c
+++ b/src/hid/step/object3d_step.c
@@ -1,11 +1,12 @@
 #include <stdio.h>
 //#include <stdlib.h>
-//#include <stdbool.h>
+#include <stdbool.h>
 
 #include <glib.h>
 
 #include "hid/common/quad.h"
 #include "hid/common/vertex3d.h"
+#include "hid/common/contour3d.h"
 #include "hid/common/appearance.h"
 #include "hid/common/face3d.h"
 //#include "hid/common/edge3d.h"
